sceptre/hooks/cmd.py

```diff
@@ -19,10 +19,15 @@ def run(self):
         :raises: subprocess.CalledProcessError
         """
         envs = self.stack.connection_manager.create_session_environment_variables()
-        try:
-            subprocess.check_call(self.argument, shell=True, env=envs)
-        except TypeError:
-            raise InvalidHookArgumentTypeError(
-                'The argument "{0}" is the wrong type - cmd hooks require '
-                "arguments of type string.".format(self.argument)
-            )
+
+        if self.argument is None:
+            raise InvalidHookArgumentTypeError()
+
+        if isinstance(self.argument, str) or self.argument is None:
```

Comment:
doesn't the previous condition already check `self.argument is None`?

Reply:
Good catch. I added the previous condition but didn't change it here.